source/includes/container.md

```diff
@@ -740,9 +745,16 @@ Available in Tutum's **STREAM API**
 
 `GET /v1/container/(uuid)/exec/`
 
-### Query Parameters
+### Path Parameters
 
 Parameter | Description
 --------- | -----------
 uuid | The UUID of the container where the command will be executed
+
+### Query Parameters
+
+Parameter | Description
+--------- | -----------
 command | Command to be executed (default: `sh`)
+user | Your Tutum username
```

Comment:
I'd remove the `user` and `token` query parameters from here, as they apply to all stream API endpoints